src/components/Footer/Footer.js

```diff
@@ -0,0 +1,38 @@
+import { Box, Link, Text, HStack } from "@chakra-ui/react";
+import { BsGithub } from "react-icons/bs";
+import React from "react";
+
+function Footer() {
+    const currentYear = new Date().getFullYear();
+    return (
+        <Box
+            bgColor={"teal"}
+            width={"100%"}
+            p={4}
+            textAlign="center"
+            paddingTop={5}
+            paddingBottom={5}
+            color="#E7E6DA"
+        >
+            <Text fontSize="16" mb={2}>
+                SQL Online Editor - Copyright © {currentYear}
```

Comment:
Highlight the `janvi01` link, in spite of `a` tag you can use `link` tag

```suggestion
                SQL Online Editor - Copyright © {currentYear} | Created by <a href="https://github.com/janvi01" target="_blank">janvi01</a>
```